pandas/tests/series/conftest.py

```diff
@@ -3,16 +3,6 @@
 import pandas.util.testing as tm
 
 
-@pytest.fixture
-def datetime_series():
-    """
-    Fixture for Series of floats with DatetimeIndex
-    """
-    s = tm.makeTimeSeries()
-    s.name = "ts"
-    return s
```

Comment:
IIRC the name of this has caused some confusion, since i think it is a numeric Series with a DatetimeIndex.  might be worth giving a more informative name

Reply:
Good point, I wouldn't want to tackle that in this PR though. I'm just moving it to a different location so I have access to it in the tests I'm changing.